### ISSUE TITLE
[cmc] missing tests of pybindings, python example fails (Trac #1136)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1130]
cmc does not have any tests for its pybindings, the folder resources/test/ is empty.

a script that looks like a test exists in resources/script/test_cmc_service.py, but it fails on my machine with this error message:
Traceback (most recent call last):
  File "cmc/resources/scripts/test_cmc_service.py", line 31, in <module>
    daughters = c.Propagate(particle, icetray.I3Frame() )
Boost.Python.ArgumentError: Python argument types in
    I3PropagatorService.Propagate(I3CascadeMCService, I3Particle, I3Frame)
did not match C++ signature:
    Propagate(boost::shared_ptr<I3PropagatorServiceWrapper> {lvalue}, I3Particle {lvalue}, boost::shared_ptr<I3PropagatorService::DiagnosticMap>)
    Propagate(I3PropagatorService {lvalue}, I3Particle {lvalue}, boost::shared_ptr<I3PropagatorService::DiagnosticMap>)

Migrated from https://code.icecube.wisc.edu/ticket/1136
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:35", 
    "description": "cmc does not have any tests for its pybindings, the folder resources/test/ is empty.\n\na script that looks like a test exists in resources/script/test_cmc_service.py, but it fails on my machine with this error message:\nTraceback (most recent call last):\n  File \"cmc/resources/scripts/test_cmc_service.py\", line 31, in <module>\n    daughters = c.Propagate(particle, icetray.I3Frame() )\nBoost.Python.ArgumentError: Python argument types in\n    I3PropagatorService.Propagate(I3CascadeMCService, I3Particle, I3Frame)\ndid not match C++ signature:\n    Propagate(boost::shared_ptr<I3PropagatorServiceWrapper> {lvalue}, I3Particle {lvalue}, boost::shared_ptr<I3PropagatorService::DiagnosticMap>)\n    Propagate(I3PropagatorService {lvalue}, I3Particle {lvalue}, boost::shared_ptr<I3PropagatorService::DiagnosticMap>)\n", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067215093672", 
    "component": "combo simulation", 
    "summary": "[cmc] missing tests of pybindings, python example fails", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-17T20:41:30", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150818T20:35:35: hdembinski changed milestone from "" to "IceSim 4.2"

**Comment by icecube-trac** [comment]
Trac update at 20150820T20:57:25: 
* nega changed status from "new" to "assigned"
* nega changed owner from "syoshida@hepburn.s.chiba-u.ac.jp" to "syoshida"

**Comment by icecube-trac** [comment]
Trac update at 20150922T01:40:00: olivas changed milestone from "IceSim 4.2" to "IceSim 5.0"

**Comment by icecube-trac** [comment]
Trac update at 20160119T03:01:13: 
* olivas commented:

> We're going to work on this next week.  Pushing to IceSim 5.1.


* olivas changed milestone from "IceSim 5.0" to "IceSim 5.1"

**Comment by icecube-trac** [comment]
Trac update at 20170117T18:46:53: 
* olivas commented:

> It's mine until I find a new home for this project.


* olivas changed owner from "syoshida" to "olivas"
* olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170118T19:03:44: 
* olivas changed status from "accepted" to "assigned"
* olivas changed owner from "olivas" to ""

**Comment by icecube-trac** [comment]
Trac update at 20170405T16:21:39: 
* olivas changed owner from "" to "olivas"
* olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170420T18:23:39: 
* olivas changed status from "accepted" to "closed"
* olivas commented:

> In changeset 155158:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="155158"
> fixed example script.  closes #1130.
> ```


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:35: 
* olivas changed milestone from "IceSim 5.1" to ""
* olivas commented:

> Milestone IceSim 5.1 deleted
